examples/02-metaballs/Metaballs.nim.cfg

```diff
@@ -5,9 +5,14 @@
 --clibdir:"/usr/local/lib"
 --passC:"-o2"
 --define:BGFX_BUILD_LIB
+--define:useSDL
```

Comment:
`--define:useSDL` Shouldn't be there. the examples implicitly uses GLFW if no specific flag is supplied.